src/components/utils/__tests__/formatValue.spec.ts

```diff
@@ -47,7 +47,7 @@ describe('formatValue', () => {
     ).toEqual('1.234.567-89');
   });
 
-  it('should handle empty decimal separator', () => {
+  it.skip('should handle empty decimal separator', () => {
     expect(
       formatValue({
         value: '1234567-89',
```

Comment:
How user interprets this number 123-89? Again I understand how to fix this test case.. Please update comments

Reply:
This test is to confirm nothing happens if decimal separator is empty string, so it applies the group separator but doesn't try to format the remaining values.